lib/all_unique.js

```diff
@@ -1,14 +1,20 @@
 module.exports = {
   all_unique: function(str){
-    // Your code goes here.
+    var abc_string = str.sort();
+    var abc_array = abc_string.split("");
+    for var x = 0; x < abc_array.length ; x++){
+      if abc_arr[x] = abc_arr[x + 1]
```

Comment:
Remember that in JS, conditional blocks are written like so:
```javascript
if (conditional) {
  code to run
}
```

Please also remember than `=` is an assignment operator, not a comparison.